vitessce/_version.py

```diff
@@ -1,6 +1,6 @@
 # Module version
-py_version_info = (1, 0, 6)
-js_version_info = (0, 1, 11)
+py_version_info = (1, 0, 7)
+js_version_info = (0, 1, 12)
```

Comment:
Not essential for this PR, but would it make sense to read from package.json here? And could we bump the JS version up so it tracks python?

Reply:
Those are good ideas, I will make an issue